modules/overrides/http.go

```diff
@@ -0,0 +1,115 @@
+package overrides
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/go-kit/log/level"
+	jsoniter "github.com/json-iterator/go"
+	"github.com/opentracing/opentracing-go"
+	"github.com/pkg/errors"
+	"github.com/weaveworks/common/user"
+
+	"github.com/grafana/tempo/pkg/api"
+	"github.com/grafana/tempo/pkg/util/log"
+)
+
+// OverridesHandler is a http.HandlerFunc to return user configured overrides
+func (o *userConfigOverridesManager) OverridesHandler(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+	span, ctx := opentracing.StartSpanFromContext(ctx, "userConfigOverridesManager.OverridesHandler")
+	defer span.Finish()
```

Comment:
Do you want to set the span error if there is one?

Reply:
I have addressed this in my last commit by setting the error in `handleError`